app/controllers/chatfuel_controller.rb

```diff
@@ -53,6 +53,26 @@ def json_response
           }
         }
       ]
+    elsif @question_answer && !@next_question_answer
+      topic = @text_component.topic
+      messages =  [
+          {
+          attachment: {
+            payload: {
+              template_type: "button",
+              text: "#{content}",
+              buttons: [
+                {
+                  type: "show_block",
+                  block_name: "cont_" + topic.name,
+                  title: "Zurück",
+                }
+            ]
+            },
+            type: "template"
+          }
+        }
+      ]
     else
```

Comment:
I cannot see a `validates :topic, presence: true` in https://github.com/roschaefer/story.board/blob/master/app/models/text_component.rb so I guess it's possible for a text component not to have a topic. It's good that you ask btw. Some developers add `nil` checks when it can never happen - that' what I call `Angstcode` :fearful: